ccpp/suites/suite_HMTW_2020_mp2.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This suite looks like it is derived from GFS_v15p2. Is that right? If not, could you edit the PR description to describe the purpose of the suite (e.g. operational GFS suite with NSSL MP substituted for GFDL MP)?

Reply:
PR comment has been edited.